core/webif.ml

```diff
@@ -38,7 +38,10 @@ struct
 
     let func =
       match fvs with
-      | `Record [] -> `FunctionPtr (int_of_string fname, None)
+      | `Record [] -> let i_fname = int_of_string fname in
```

Comment:
I would add a comment here explaining the reason for the conditional behavior: basically IMO the reason is there is an artificial distinction between primitive functions (builtin functions) and function pointers (functions defined as part of a Links program), and this is the point where we have to do something about it in order for attempts to remotely call primitive functions to work properly.